docs/conf.py

```diff
@@ -145,16 +145,16 @@
 linkcheck_ignore = [r"https://doi.org/\d+",
                     r"https://riot.im/\d+",
                     r"https://github.com/\d+",
-                    r"http://docs.sunpy.org/\d+"]
+                    r"https://docs.sunpy.org/\d+"]
 linkcheck_anchors = False
 
 # This is added to the end of RST files - a good place to put substitutions to
 # be used globally.
 rst_epilog = """
 .. SunPy
-.. _SunPy: http://sunpy.org
-.. _`SunPy mailing list`: http://groups.google.com/group/sunpy
-.. _`SunPy dev mailing list`: http://groups.google.com/group/sunpy-dev
+.. _SunPy: s//sunpy.org
```

Comment:
Is this right?

Reply:
Not sure what happened here. I will fix that.